src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.RequestSeparator.cs

```diff
@@ -43,5 +43,34 @@ public void tokens_on_request_separator_nodes_are_parsed_into_request_separator(
                                     .Should().ContainSingle<HttpRequestSeparatorNode>()
                                     .Which.Text.Should().Be("### Slow Response (Json)");
         }
+
+        [Fact]
+        public void tokens_on_request_separator_nodes()
```

Comment:
The test name should be more descriptive of the expectation.

Reply:
Done